### ISSUE TITLE
Update meteor-spk to Meteor, Node 12.x and newest MongoDB 4.x ?

### ISSUE THREAD

**Issue by xet7** [sandstorm-io/meteor-spk#35]
@zenhack can you look at this?

Currently Wekan uses Meteor 1.9, Node 12.15.0 and newest MongoDB on Docker (although any MongoDB 3.x or 4.x works). Wekan Snap still uses MongoDB 3.2.22 because I have not got MongoDB 4.x working in Snap yet.

Newest Wekan for Sandstorm is v3.72, after that I was not able to build new Wekan .spk package. Newest Wekan for other platforms is Wekan v3.79.

Could this meteor-spk be updated to newest versions?

Currently I have errors because newest fibers is not in meteor-spk, so when I make Wekan spk package, Wekan does not start, similar like [when building Sandstorm](https://github.com/sandstorm-io/sandstorm/issues/3177) so that fibers.node is not found, this is log of from Wekan grain:

```
** SANDSTORM SUPERVISOR: Starting up grain. Sandbox type: userns
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
** SANDSTORM SUPERVISOR: Grain no longer in use; shutting down.
** SANDSTORM SUPERVISOR: Starting up grain. Sandbox type: userns
** Starting Mongo...
2020-02-11T22:22:09.855+0000 I STORAGE  Engine custom option: log=(prealloc=false,file_max=200KB)
about to fork child process, waiting until server is ready for connections.
forked process: 10
child process started successfully, parent exiting
** Starting Meteor...
## There is an issue with `node-fibers` ##
`/programs/server/node_modules/fibers/bin/linux-x64-57-glibc/fibers.node` is missing.

Try running this to fix the issue: node /programs/server/node_modules/fibers/build
Error: Cannot find module '/programs/server/node_modules/fibers/bin/linux-x64-57-glibc/fibers'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/programs/server/node_modules/fibers/fibers.js:14:39)
    at Module._compile (module.js:653:30)
    at Module.Mp._compile (/programs/server/runtime.js:50:23)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at Module.Mp.load (/programs/server/runtime.js:15:31)
/programs/server/node_modules/fibers/fibers.js:23
		throw new Error('Missing binary. See message above.');
		^
Error: Missing binary. See message above.
    at Object.<anonymous> (/programs/server/node_modules/fibers/fibers.js:23:9)
    at Module._compile (module.js:653:30)
    at Module.Mp._compile (/programs/server/runtime.js:50:23)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at Module.Mp.load (/programs/server/runtime.js:15:31)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
** HTTP-BRIDGE: App server exited with status code: 1
```
When I tried to build meteor-spk, there was git submodules of some commit of MongoDB 3.0 from 2014 and also Kenton's fork of MongoDB. Building it failed on Ubuntu 19.10 64bit. Then I tried to update those submodules to master branches, and got this error:
```
$ make
**** Building MongoDB 3.0 ...
cd mongo && scons -j6 mongod --disable-warnings-as-errors
scons: Reading SConscript files ...
  File "/home/wekan/repos/meteor-spk/mongo/SConstruct", line 348
    add_option(f'use-system-{name}',
                                  ^
SyntaxError: invalid syntax

make: *** [Makefile:18: mongo/mongod] Error 2
```
Do we still need some fork of MongoDB ?

**Comment by zenhack** [comment]
I assume the syntax error is because it's trying to run the SConstruct file with python 2, and master is using some python 3 only features.

I remember hitting some build errors with mongo which I managed to patch around, but got stuck somewhere else, at this point I forget where.

**Comment by xet7** [comment]
Meanwhile, using old version of Meteor and Node.js, I published newest Wekan v3.79 to [experimental](https://apps.sandstorm.io/app/m86q05rdvj14yvn78ghaxynqz7u2svw6rnttptxx49g1785cdv1h?experimental=true) and also [updated my Sandstorm Wekan build script](https://github.com/wekan/wekan/blob/master/releases/release-sandstorm.sh).

**Comment by kentonv** [comment]
I've released meteor-spk version 0.5.1 based on Meteor 1.10.1 and Node 12.
